bitnami/logstash/templates/sts.yaml

```diff
@@ -98,6 +101,9 @@ spec:
           {{- if .Values.extraVolumeMounts }}
             {{- include "common.tplvalues.render" ( dict "value" .Values.extraVolumeMounts "context" $ ) | nindent 12 }}
           {{- end }}
+          {{- if .Values.lifecycle }}
+          lifecycle: {{- toYaml .Values.lifecycle | nindent 12 }}
```

Comment:
Could you common.values.tplrender here as well?

Reply:
Done.